src/libs/Navigation/linkTo.js

```diff
@@ -60,8 +60,12 @@ export default function linkTo(navigation, path, type) {
 
     // If action type is different than NAVIGATE we can't change it to the PUSH safely
     if (action.type === 'NAVIGATE') {
-        // If this action is navigating to the report screen and the top most navigator is different from the one we want to navigate - PUSH
-        if (action.payload.name === NAVIGATORS.CENTRAL_PANE_NAVIGATOR && getTopmostReportId(root.getState()) !== getTopmostReportId(state)) {
+        // In case if type is 'FORCED_UP' we ensure that we need to replace current screen to the provided
+        if (type === 'FORCED_UP') {
```

Comment:
Isn't it possible to use the existing "UP" type which does the same below?

Reply:
Or (also?), all these string values could be just extracted to the `CONST.js` file.